README.md

```diff
@@ -39,31 +39,31 @@ $ brew remove eosio
 ```
 #### Ubuntu 18.04 Debian Package Install
 ```sh
-$ wget https://github.com/eosio/eos/releases/download/v1.5.0-rc2/eosio_1.5.0-rc2-1-ubuntu-18.04_amd64.deb
-$ sudo apt install ./eosio_1.5.0-rc2-ubuntu-18.04_amd64.deb
+$ wget https://github.com/eosio/eos/releases/download/v1.5.0/eosio_1.5.0-1-ubuntu-18.04_amd64.deb
+$ sudo apt install ./eosio_1.5.0-ubuntu-18.04_amd64.deb
```

Comment:
These also need the release in them, so it becomes `eosio_1.5.0-1-ubuntu-18.04_amd64.deb`

Reply:
just to be clear the `wget` lines were correct but the `apt install` lines were not, right? (if so, updated)

Reply:
thats correct